meson_options.txt

```diff
@@ -0,0 +1,19 @@
+option('tracy_enable', type : 'boolean', value : true, description : 'Enable profiling')
+option('tracy_on_demand', type : 'boolean', value : false, description : 'On-demand profiling')
+option('tracy_callstack', type : 'boolean', value : false, description : 'Enfore callstack collection for tracy regions')
+option('tracy_no_callstack', type : 'boolean', value : false, description : 'Disable all callstack related functionality')
+option('tracy_no_callstack_inlines', type : 'boolean', value : false, description : 'Disables the inline functions in callstacks')
+option('tracy_only_localhost', type : 'boolean', value : false, description : 'Only listen on the localhost interface')
+option('tracy_no_broadcast', type : 'boolean', value : false, description : 'Disable client discovery by broadcast to local network')
+option('tracy_only_ipv4', type : 'boolean', value : false, description : 'Tracy will only accept connections on IPv4 addresses (disable IPv6)')
+option('tracy_no_code_transfer', type : 'boolean', value : false, description : 'Disable collection of source code')
+option('tracy_no_context_switch', type : 'boolean', value : false, description : 'Disable capture of context switches')
+option('tracy_no_exit', type : 'boolean', value : false, description : 'Client executable does not exit until all profile data is sent to server')
+option('tracy_no_sampling', type : 'boolean', value : false, description : 'Disable call stack sampling')
+option('tracy_no_verify', type : 'boolean', value : false, description : 'Disable zone validation for C API')
+option('tracy_no_vsync_capture', type : 'boolean', value : false, description : 'Disable capture of hardware Vsync events')
+option('tracy_no_frame_image', type : 'boolean', value : false, description : '"Disable the frame image support and its thread')
+option('tracy_delayed_init', type : 'boolean', value : false, description : 'Enable delayed initialization of the library (init on first call)')
+option('tracy_manual_lifetime', type : 'boolean', value : false, description : 'Enable the manual lifetime management of the profile')
+option('tracy_fibers', type : 'boolean', value : false, description : 'Enable fibers support')
+option('tracy_shared_libs', type : 'boolean', value : false, description : 'Builds Tracy as a shared object')
```

Comment:
FWIW: when defining these options, you don't need to prefix it with the project name.

Meson has a different object type for variables vs. options and options are per-subproject. So these would now be `meson setup builddir -Dtracy:tracy_enable=false` and suchlike.